shacl12-profiling/index.html

```diff
@@ -438,6 +438,50 @@
 
 		<section id="introduction">
 			<h2>Introduction</h2>
+			<section id="what-is">
+				<h3>What is profiling?</h3>
+				<p>
+					Profiling is the act of creating a "profile" of something.
+				</p>
+				<p>
+					Generically, in English, a "profile" of something is:
+				</p>
+				<p style="padding: 0 50px 0 50px;">
+					The outline of a physical object or feature, or a representation of this
+					<br /><br />- Oxford English dictionary, use of the word "profile" since the 17th century
+				</p>
+				<p>
+					Within the world of data, a derived definition of "profile" consistent with the above is:
+				</p>
+				<p>
+					A summary or an extraction
+				</p>
+				<p>
+					In this definition, the essence of the English word is retained since a summary or extraction of/from a data object <em>may</em> be an outline of it, for example a 2D representation of a 3D spatial object or a statistical summary of a dataset with lots of parts.
+				</p>
+				<p>
+					SHACL, by definition, constrains (RDF) data, and thus any data valid according to a shapes graph will be a profile of the data graph that was validated. In the case of a shapes graph validating all elements of a data graph, the valid data will be a "null" profile of the data graph, that is identical to it.
+				</p>
+				<p>
+					The W3C's <em>Profiles Vocabulary</em> [[dx-prof]] has defined "data profiling" profiling in the context of <em>specifications</em> or <em>data specifications</em>:
+				</p>
+				<p style="padding: 0 50px 0 50px;">
+					<em>A data specification that constrains, extends, combines, or provides guidance or explanation about the usage of other data specifications.</em>
+				</p>
+				<p>
+					If a shapes graph is taken to be a "data specification", and it can sensibly be, then in addition to data valid according to a shapes graph being a profile of the data graph validated, the shapes graph itself is a profile of the data model used for a data graph.
```

Comment:
```suggestion
					If a shapes graph is taken to be a "data specification" (and it can sensibly be), then, in addition to data valid according to a shapes graph being a profile of the data graph validated, the shapes graph itself is a profile of the data model used for the data graph.
```

Reply:
Applied in https://github.com/w3c/data-shapes/pull/489/commits/adb648c749af114ec3c38437e3af9b4c1538cc3c